README.md

```diff
@@ -47,7 +47,7 @@ file.
 Deploy the demo to your AWS account using [AWS CDK](https://aws.amazon.com/cdk/).
 
 ```bash
-cdk deploy
+cdk deploy --all
 ```
 
 The command `cdk deploy` will first build the products maven project using a docker build image with all the required
```

Comment:
Not really related to this demo, just something weird I met. I ran `artillery` on AL2, with version info:
```
VERSION INFO:

Artillery: 2.0.19
Node.js:   v16.20.2
OS:        linux
```

I met `ReferenceError: ReadableStream is not defined`, and I have to add `const { types, ReadableStream } = require('node:util')` to `~/.nvm/versions/node/v16.20.2/lib/node_modules/artillery/node_modules/undici/lib/web/fetch/response.js`

Maybe I am using an old nodejs?